docs/src/main/asciidoc/gradle-tooling.adoc

```diff
@@ -501,6 +501,18 @@ quarkus {
 
 ====
 
+== Running integration tests
+
+Quarkus integration tests (annotated with `@QuarkusIntegrationTest`) will run on the artifact produced by Quarkus.
+Those tests can be placed in a `src/intTest/java` directory and executed using:
+
+[source, bash]
+----
+./gradlew quarkusIntTest
+----
+
+This task depends on `quarkusBuild`. The final artifact will be produced before running tests.
+
```

Comment:
Does `quarkusBuild` also run the `check` task? If so, maybe you should mention that the unit tests will also be executed?

Maybe not necessary, but just a thought.

Reply:
Currently `quarkusBuild` is not linked to any `test` task

Reply:
Should it (or the `intTest` task) be?

Reply:
I will add a dependency between `quarkusIntTest` and `check`